<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="32499" started-at="2019-08-14T16:00:41Z" finished-at="2019-08-14T16:01:14Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="32499" started-at="2019-08-14T16:00:41Z" finished-at="2019-08-14T16:01:14Z">
      <class name="guru99.gittest.AppTest">
        <test-method status="PASS" signature="CenterSelection()[pri:0, instance:guru99.gittest.AppTest@25af5db5]" name="CenterSelection" is-config="true" duration-ms="8" started-at="2019-08-14T16:00:30Z" finished-at="2019-08-14T16:00:30Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- CenterSelection -->
        <test-method status="PASS" signature="LoginDetails()[pri:0, instance:guru99.gittest.AppTest@25af5db5]" name="LoginDetails" is-config="true" duration-ms="0" started-at="2019-08-14T16:00:30Z" finished-at="2019-08-14T16:00:30Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- LoginDetails -->
        <test-method status="PASS" signature="startUp()[pri:0, instance:guru99.gittest.AppTest@25af5db5]" name="startUp" is-config="true" duration-ms="11315" started-at="2019-08-14T16:00:30Z" finished-at="2019-08-14T16:00:41Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- startUp -->
        <test-method status="PASS" signature="beforeTest()[pri:0, instance:guru99.gittest.AppTest@25af5db5]" name="beforeTest" is-config="true" duration-ms="0" started-at="2019-08-14T16:00:41Z" finished-at="2019-08-14T16:00:41Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeTest -->
        <test-method status="FAIL" signature="MemberCreation()[pri:1, instance:guru99.gittest.AppTest@25af5db5]" name="MemberCreation" duration-ms="32472" started-at="2019-08-14T16:00:41Z" finished-at="2019-08-14T16:01:14Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Error Message] but found []]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Error Message] but found []
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:453)
at org.testng.Assert.assertEquals(Assert.java:463)
at guru99.gittest.Membershipbill_productSelection.SaveBill(Membershipbill_productSelection.java:78)
at guru99.gittest.AppTest.MemberCreation(AppTest.java:51)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- MemberCreation -->
        <test-method status="PASS" signature="afterTest()[pri:0, instance:guru99.gittest.AppTest@25af5db5]" name="afterTest" is-config="true" duration-ms="0" started-at="2019-08-14T16:01:14Z" finished-at="2019-08-14T16:01:14Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterTest -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:guru99.gittest.AppTest@25af5db5]" name="tearDown" is-config="true" duration-ms="0" started-at="2019-08-14T16:01:14Z" finished-at="2019-08-14T16:01:14Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- guru99.gittest.AppTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
